### DISCUSSION TITLE
Explanation/Resource required on _pmv_ppd_optimized function

### DISCUSSION THREAD

**Discussion by NeelakshSharma** [CenterForTheBuiltEnvironment/pythermalcomfort#110] [Q&A]
Hi all, I am trying to study the code on how the pmv ppd indices are calculated. For this I came across the function `_pmv_ppd_optimzied()` . Although I know the pmv index equation as per ASHRAE standard, but I was stuck on some lines of the code. To be specific: `t_cla = taa + (35.5 - tdb) / (3.5 * icl + 0.1)`.
It would be really helpful if someone could provide the source from where the calculation was translated to the Python code.
Thanks.

**Comment by FedericoTartarini** [comment]
Hi, the function pmv_ppd_otimised is used to speed up the code of the pmv calculation. I am sorry but I do not understand your question. The equation you are referring to is taken from the original PMV code.
